contracts/SiphonDepositor.sol

```diff
@@ -0,0 +1,185 @@
+    function withdraw(uint256 _pid, uint256 _amount) external returns(bool);
+    function poolInfo(uint256) external view returns(address,address,address,address,address, bool);
+}
+
+// prettier-ignore
+interface IBaseRewardPool {
+    function getReward(address _account, bool _claimExtras) external returns(bool);
+}
+
+// prettier-ignore
+interface ICvx is IERC20 {
+    function INIT_MINT_AMOUNT() external view returns (uint256);
+    function minterMinted() external view returns (uint256);
+    function reductionPerCliff() external view returns (uint256);
+    function totalCliffs() external view returns (uint256);
+    function EMISSIONS_MAX_SUPPLY() external view returns (uint256);
+}
+
+// prettier-ignore
+interface IrCvx is IERC20 {
+    function mint(address,uint256) external; 
+    function burn(address,uint256) external; 
+}
+
+contract SiphonDepositor is Ownable {
+    using AuraMath for uint256;
+    using SafeERC20 for IERC20;
+    using SafeERC20 for ICvx;
+
+    IERC20 public immutable lpToken;
+    IERC20 public immutable crv;
+    IBooster public immutable booster;
+    ICvx public immutable cvx;
+    IrCvx public immutable rCvx;
+    IAuraLocker public immutable auraLocker;
+    uint256 public immutable pid;
+
+    /**
+     * @dev Penalty basis points 2500 == 25%
+     */
+    uint256 public immutable penaltyBp;
+
+    constructor(
+        IERC20 _lpToken,
+        IERC20 _crv,
+        IBooster _booster,
+        ICvx _cvx,
+        IrCvx _rCvx,
+        IAuraLocker _auraLocker,
+        uint256 _pid,
+        uint256 _penaltyBp
+    ) {
+        lpToken = _lpToken;
+        crv = _crv;
+        booster = _booster;
+        cvx = _cvx;
+        rCvx = _rCvx;
+        auraLocker = _auraLocker;
+        pid = _pid;
+        penaltyBp = _penaltyBp;
+    }
+
+    /**
+     * @dev deposit "lpTokens" into the booster
+     */
+    function deposit() external {
+        lpToken.approve(address(booster), type(uint256).max);
+        uint256 bal = lpToken.balanceOf(address(this));
+        booster.deposit(pid, bal, true);
+    }
+
+    /**
+     * @dev withdraw "lpTokens" from the booster
+     */
+    function withdraw(uint256 _amount) external {
+        booster.withdraw(pid, _amount);
+    }
+
+    /**
+     * @dev Siphon AURA tokens by depositing BAL into the Booster
+     *      and then calling earmark rewards which will send the BAL
+     *      to the siphon pools BaseRewardPool
+     * @param _amount Amount of BAL that is being bridged from the L2 to cover the
+     *                Incentive amount that was paid out.
+     *                We assume the total incentives that have been paid out are equal
+     *                to the MaxFees on the Booster which is 2500/10000 (25%)
+     */
+    function siphon(uint256 _amount) external {
+        // TODO: only callable by admin or lzEndpoint
+
+        uint256 amount = (_amount * 10000) / 2500;
```

Comment:
Will likely have to manually make sure the L2 rate and the L1 rate are in sync. But for this I've updated it to grab the fees directly from the booster so this will always be correct.